third_party/2and3/pynamodb/models.pyi

```diff
@@ -0,0 +1,82 @@
+from .exceptions import DoesNotExist
+from typing import Any, Dict, Generic, Iterable, Iterator, List, Optional, Tuple, Type, TypeVar, Text, Union
+
+log: Any
+
+class DefaultMeta: ...
+
+class ResultSet(Iterable):
+    results: Any
+    operation: Any
+    arguments: Any
+    def __init__(self, results, operation, arguments) -> None: ...
+    def __iter__(self): ...
+
+class MetaModel(type):
+    def __init__(self, name: Text, bases: Tuple[type, ...], attrs: Dict[Any, Any]) -> None: ...
+
+_T = TypeVar('_T', bound='Model')
+KeyType = Union[Text, bytes, float, int, Tuple]
+
+class Model(metaclass=MetaModel):
+    DoesNotExist = DoesNotExist
+    attribute_values: Dict[Text, Any]
+    def __init__(self, hash_key: Optional[KeyType] = ..., range_key: Optional[Any] = ..., **attrs) -> None: ...
+    @classmethod
+    def has_map_or_list_attributes(cls: Type[_T]) -> bool: ...
+    @classmethod
+    def batch_get(cls: Type[_T], items: Iterable[Union[KeyType, Iterable[KeyType]]], consistent_read: Optional[bool] = ..., attributes_to_get: Optional[List[Text]] = ...) -> Iterator[_T]: ...
```

Comment:
Should `attributes_to_get` be a Sequence instead? List is invariant, so this annotation will make mypy reject `List[str]` in Python 2.

From looking at the code (https://github.com/pynamodb/PynamoDB/blob/master/pynamodb/models.py#L249), the argument is actually ignored, which seems like a bug in the library.